hoomd/md/methods/methods.py

```diff
@@ -691,7 +691,7 @@ class Langevin(Method):
     where :math:`\vec{\tau}_\mathrm{C} = \vec{\tau}_\mathrm{net}`,
     :math:`\gamma_r^i` is the i-th component of the rotational drag coefficient
     (`gamma_r`), :math:`\tau_\mathrm{R}^i` is a component of the uniform random
-    the torque, :math:`\vec{L}` is the particle's angular momentum and :math:`I`
+    the torque, :math:`\vec{L}` is the particle's angular velocity and :math:`I`
```

Comment:
```suggestion
    torque, :math:`\vec{L}` is the particle's angular velocity and :math:`I`
```
Not related to units or variables, but I think there is an extra "the" in that sentence.